### PR TITLE
Added the optimize_expressjs_apps.md blogpost

### PR DESCRIPTION
**Pull request by bogdanripa**

### CODE REVIEW

File: content/posts/optimize_expressjs_apps.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+title: Ultimate Guide to Optimizing Website and Application Development with Express.js
+date: 2024-08-22
+tags:
+  - Learning
+author: Bogdan Ripa
+linkedIn: https://www.linkedin.com/in/bogdanripa
+thumbnail: /images/fosdem-2024.webp
+preview: "Discover best practices for optimizing Express.js development with serverless architecture and CDN deployment on Genezio"
+# meta data start
+description: "RLearn how to optimize your Express.js development by implementing serverless architecture and CDN deployment. Explore how Genezio’s platform enhances performance, scalability, and cost-efficiency, making it easier to build and deploy modern web applications"
```
Comment (andreia-oca) [inline content/posts/optimize_expressjs_apps.md:11]:
```suggestion
description: "Learn how to optimize your Express.js development by implementing serverless architecture and CDN deployment. Explore how Genezio’s platform enhances performance, scalability, and cost-efficiency, making it easier to build and deploy modern web applications"
```

File: content/posts/optimize_expressjs_apps.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+title: Ultimate Guide to Optimizing Website and Application Development with Express.js
+date: 2024-08-22
+tags:
+  - Learning
+author: Bogdan Ripa
+linkedIn: https://www.linkedin.com/in/bogdanripa
+thumbnail: /images/fosdem-2024.webp
```
Comment (andreia-oca) [inline content/posts/optimize_expressjs_apps.md:8]: We have to change this to use a branding-aligned thumbnail image.

**Review by andreia-oca — APPROVED** [review]
LGTM!

File: content/posts/optimize_expressjs_apps.md (hunk 2 above)
Comment (bogdanripa) [reply content/posts/optimize_expressjs_apps.md:8]: how?

**Pull request merged**